source/cloud-native-platform/path-to-live/_front-door-config.md

```diff
@@ -44,6 +45,7 @@
     name             = "app-name"
     custom_domain    = "your-app.service.gov.uk"
     backend_domain   = ["firewall-prod-int-palo-sdsprod.uksouth.cloudapp.azure.com"]
+    add_txt_record   = true
```

Comment:
shouldn't we just do this by default? or is there a problem with that?

Reply:
just sent you PM on slack